part-spec/sensor.json

```diff
@@ -223,6 +223,175 @@
                     }
                 }
             }
+        },
+        {
+            "title": "magnetic sensor",
+            "type": "object",
+            "required": [
+                "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "type": {
+                    "description": "method by which magnetism is detected",
+                    "enum": [
+                        "gmr",
+                        "hall effect"
+                    ],
+                    "type": "string"
+                },
+                "outputType": {
+                    "description": "measurement output type",
+                    "enum": [
```

Comment:
examples, not enum - are there any other options?

Reply:
done